src/openrct2/audio/Audio.cpp

```diff
@@ -169,12 +169,22 @@ void audio_populate_devices()
     devices.insert(devices.begin(), defaultDevice);
 #endif
 
-    gAudioDeviceCount = static_cast<int32_t>(devices.size());
-    gAudioDevices = Memory::AllocateArray<audio_device>(gAudioDeviceCount);
-    for (int32_t i = 0; i < gAudioDeviceCount; i++)
+    if (devices.size())
     {
-        auto device = &gAudioDevices[i];
-        String::Set(device->name, sizeof(device->name), devices[i].c_str());
+        gAudioDeviceCount = static_cast<int32_t>(devices.size());
+        gAudioDevices = Memory::AllocateArray<audio_device>(gAudioDeviceCount);
+        for (int32_t i = 0; i < gAudioDeviceCount; i++)
+        {
+            auto device = &gAudioDevices[i];
+            String::Set(device->name, sizeof(device->name), devices[i].c_str());
+        }
+    }
+    else
+    {
+        gAudioDeviceCount = 1;
+        gAudioDevices = Memory::AllocateArray<audio_device>(gAudioDeviceCount);
+        auto device = &gAudioDevices[0];
+        String::Set(device->name, sizeof(device->name), "No Audio Device");
```

Comment:
I'm not sure if faking an audio device is a good idea, unless the alternative is dozens of checks elsewhere.